prov/efa/src/rdm/efa_rdm_cq.c

```diff
@@ -371,10 +371,15 @@ static void efa_rdm_cq_handle_recv_completion(struct efa_ibv_cq *ibv_cq, struct
 	 * QP and we cannot cancel that.
 	 */
 	if (OFI_UNLIKELY(ep->use_zcpy_rx && efa_rdm_pkt_type_is_rtm(pkt_type))) {
+		char peer_host_id_str[EFA_HOST_ID_STRING_LENGTH + 1] = {0};
+		if (!peer->host_id || EFA_HOST_ID_STRING_LENGTH != snprintf(peer_host_id_str, EFA_HOST_ID_STRING_LENGTH + 1, "i-%017lx", peer->host_id))
+			strcpy(peer_host_id_str, "N/A");
+
 		EFA_WARN(FI_LOG_CQ,
-			"Invalid pkt type %d! Peer %d doesn't respect the request from this EP that"
-			" RTM packets must be sent to the user recv QP.\n",
-			base_hdr->type, (int)pkt_entry->addr);
+			"Packet type %d from remote peer %d (host id: %s) is not supported when zero-copy recv mode is ON."
```

Comment:
GID would be better here - if host id is not available for some reason we can still trace down the peer with GID.

Reply:
I reused `efa_rdm_write_err_msg` in the latest revision, it should have host id and gid for local & remote